static/js/domain-registration.js

```diff
@@ -76,17 +76,26 @@
                 modal.classList.add("is-visible");
 
                 const requestedDomain = document.querySelector(".js-subdomain-value");
-                const requestedDomainPreview = document.querySelectorAll(".js-modal-domain-registration-confirmation-domain-preview");
+                const requestedDomainPreview = document.querySelector(".js-modal-domain-registration-confirmation-domain-preview");
```

Comment:
@maxxcrawford Any idea why this was a `querySelectorAll` instead of a `querySelector`? As far as I can see there's only one such element in the entire codebase.

Reply:
It's in two locations, both in the `modal-domain-registration-confirmation.html` file: 
- https://github.com/mozilla/fx-private-relay/blob/main/privaterelay/templates/includes/modal-domain-registration-confirmation.html#L21
- https://github.com/mozilla/fx-private-relay/blob/main/privaterelay/templates/includes/modal-domain-registration-confirmation.html#L11

Per your demo video, it's here and here (the second instance is currently broken bc of the lack of the `querySelectorAll`) 

<img width="273" alt="image" src="https://user-images.githubusercontent.com/2692333/134451636-9a97c002-c82c-4e6b-aeb2-a7c305cd902e.png">